index.html

```diff
@@ -1779,115 +1829,218 @@ <h3>Anonymous Holder Binding</h3>
-recognize that the same holder is presenting a new selectively revealed document
-with derived proof. Note that this may just be a new unlinkable proof (derived
-proof) on the same selectively revealed information. By "privacy preserving," we
-mean that no uniquely identifiable information is added that would allow tracking
-between different verifiers that may share information amongst themselves. This
-variant does allow for the issuer to monitor usage if verifiers share
-information with the issuer.
-        </p>
-        <p>
-To furnish this capability, before creating the base proof for a document, an
-issuer generates a value known as a |pid| (prover id) which should be
-cryptographically random and at least 32 bytes long. This value is shared with
-the holder but otherwise kept secret. This value is then used in creating the
-base proof via the signing procedure in [[CFRG-Pseudonym-BBS-Signature]].
-        </p>
-        <p>
-The holder receives the document with base proof which includes the |pid| value
-from the issuer. The holder obtains a |verifier_id| associated with the verifier
-for which they intend to create a revealed document with derived proof. Using the
-procedures of [[CFRG-Pseudonym-BBS-Signature]], a cryptographic |pseudonym| value
-is generated. The derived proof value is generated via the proof generation
-procedure of [[CFRG-Pseudonym-BBS-Signature]], and this value along with the
-|pseudonym| are given to the verifier. Note that the |pid| value cannot be
-recovered from the |pseudonym|.
-        </p>
-        <p>
-When the verifier receives the revealed document with derived proof and
-|pseudonym|, they use the proof verification procedures of
-[[CFRG-Pseudonym-BBS-Signature]].
-        </p>
+        <h3>Credential-Bound Pseudonyms</h3>
+          <p>
+BBS-signed Verifiable Credentials as specified in this document allow for
+selective disclosure and unlinkable cryptographic proof artifacts. By
+"unlinkable", we mean that the
+cryptographic information in the derived proofs cannot be linked to other
+proofs nor the original signature. This implies that a verifier cannot determine
+whether a holder has presented the same credential before (with a different proof
+instantiation), nor can they assert some type of identity. Credential-bound
+pseudonyms provide a privacy preserving mechanism to allow for the limited
+linkability of a cryptographic pseudonym. Such a pseudonym may be determined
+strictly by the holder, or jointly by the holder and the verifier.
+          </p>
+          <p>
+This type of cryptographic pseudonym (cryptographic identifier/name) is
+computed from two parts. The first part, the |nym_secret|, is specified by,
+and will only be known by, the
+holder; the second part, the |nym_domain|, may be specified by either the holder
+or the verifier, and will be shared between the holder and verifier.
+Cryptographic pseudonyms computed from the same |nym_secret| but different
```

Comment:
```suggestion
An issuer binds a credential to a |nym_secret| during the issuance process. 
A holder can then compute pseudonyms from the |nym_secret| and prove to
verifiers that these pseudonyms are bound to the credential the are presenting.
Cryptographic pseudonyms computed from the same |nym_secret| but different
```

Reply:
What about something like this @Wind4Greg. Just trying to introduce the issuance part earlier.

 Perhaps also more detail to this sentence?

"An issuer binds a credential to a |nym_secret| during the issuance process using a blind signature without learning the |nym_secret|."